.gitignore

```diff
@@ -20,6 +20,7 @@ coverage
 # Folders
 _obj
 _test
+vendor
```

Comment:
For the sake of completion, this should still be undone.

Reply:
😃